packages/react/src/components/form/Checkbox/Checkbox.module.css

```diff
@@ -47,17 +60,42 @@
   position: absolute;
   left: 0;
   top: 0;
-  min-width: 44px;
-  min-height: 44px;
+  width: 2.5em;
+  height: 2.5em;
   display: inline-grid;
   grid: [input] 1fr / [input] 1fr;
   gap: var(--fds-spacing-2);
   grid-auto-flow: column;
 }
 
-.radio,
-.radio .icon {
-  border-radius: var(--fds-border_radius-full);
+.container.small {
+  min-width: var(--fds-sizing-8);
+  min-height: var(--fds-sizing-8);
+}
+
+.container.medium {
+  min-width: var(--fds-sizing-10);
+  min-height: var(--fds-sizing-10);
+}
+
+.container.large {
+  min-width: var(--fds-sizing-12);
+  min-height: var(--fds-sizing-12);
+}
+
+.container.small .control {
+  width: var(--fds-sizing-8);
+  height: var(--fds-sizing-8);
+}
+
+.container.medium .control {
+  width: var(--fds-sizing-10);
+  height: var(--fds-sizing-10);
+}
+
+.container.large .control {
+  width: var(--fds-sizing-12);
+  height: var(--fds-sizing-12);
```

Comment:
If the size is the same, can we use a variable, or `inherit` here?

Reply:
This will be change in #769 

Its `min-*` for `.container` vs fixed sizes for `.control`. Can I `inherit` from another css property?

Reply:
Right, we would need to use variables, but it's only used twice, so might be overkill 🤔